ckanext-datagov_iauthfunctions/README.rst

```diff
@@ -0,0 +1,166 @@
+.. You should enable this project on travis-ci.org and coveralls.io to make
```

Comment:
I think most of this readme isn't relevant, maybe just mention the extension in the top-level README.